iocBoot/iocLabJack/st.cmd.linux

```diff
@@ -16,15 +17,19 @@ LabJackApp_registerRecordDeviceDriver(pdbbase)
 #
 ###  LabJack support  ###
 #
-# Load one LabJack_Tx_n.cmd file for each LabJack module this ioc will support
-#
-< LabJack_T4_1.cmd
-#< LabJack_T4_2.cmd
-#< LabJack_T4_3.cmd
-#
-#< LabJack_T7_1.cmd
-#< LabJack_T7_2.cmd
-#< LabJack_T7_3.cmd
+# Load one LabJack_T7.cmd file for each LabJack module this ioc will support
+# Must set N and IP_ADDRESS differently for each one.
+
+
+epicsEnvSet("N", "1")
+epicsEnvSet("IP_ADDRESS", "gse-labjack1")
+< LabJack_T7.cmd
+epicsEnvSet("N", "2")
+epicsEnvSet("IP_ADDRESS", "gse-labjack2")
+< LabJack_T7.cmd
+#epicsEnvSet("N", "3")
+#epicsEnvSet("IP_ADDRESS", "gse-labjack3")
+#< LabJack_T7.cmd
```

Comment:
Maybe switch these to iocshLoad?

iocshLoad("LabJack_T7.com", "N=1, IP_ADDRESS=gse-labjack1")
iocshLoad("LabJack_T7.com", "N=2, IP_ADDRESS=gse-labjack2")
\#iocshLoad("LabJack_T7.com", "N=3, IP_ADDRESS=gse-labjack3")

Reply:
Thanks for the suggestion, I have made that change.